syntax="proto3";
// 配网数据结构
option java_package = "com.szlongzy.common.protobuf.dto";
// 指定模板类的名称，名称必须是有实际业务意义的
option java_outer_classname = "CheckModelVo";


/* 单个命令和属性*/
message CheckModelRequest{
  // 实际目标设备
  string  objDevId = 1;
  int64  time=2;
  //备用验证字段
  string authKey =3;

}

/* 检查模型*/
message CheckModelResponse{
  // 返回结果是否成功
  bool  result = 1;
  // 错误信息
  string  msg = 2;
  //备用验证字段
  string authKey =3;
  repeated codes data=4;

}

message codes{
  //命令编码
  repeated string code =1;
}
